diff --git a/ceph_s3_test_6d8c005.patch b/ceph_s3_test_6d8c005.patch
new file mode 100644
index 0000000..e69de29
diff --git a/s3tests_boto3/functional/__init__.py b/s3tests_boto3/functional/__init__.py
index e687805..11df7b7 100644
--- a/s3tests_boto3/functional/__init__.py
+++ b/s3tests_boto3/functional/__init__.py
@@ -119,12 +119,12 @@ def nuke_prefixed_buckets(prefix, client=None):
             objects_list = get_objects_list(bucket_name, client)
             for obj in objects_list:
                 response = client.delete_object(Bucket=bucket_name,Key=obj)
-            versioned_objects_list = get_versioned_objects_list(bucket_name, client)
-            for obj in versioned_objects_list:
-                response = client.delete_object(Bucket=bucket_name,Key=obj[0],VersionId=obj[1])
-            delete_markers = get_delete_markers_list(bucket_name, client)
-            for obj in delete_markers:
-                response = client.delete_object(Bucket=bucket_name,Key=obj[0],VersionId=obj[1])
+            # versioned_objects_list = get_versioned_objects_list(bucket_name, client)
+            # for obj in versioned_objects_list:
+            #     response = client.delete_object(Bucket=bucket_name,Key=obj[0],VersionId=obj[1])
+            # delete_markers = get_delete_markers_list(bucket_name, client)
+            # for obj in delete_markers:
+            #     response = client.delete_object(Bucket=bucket_name,Key=obj[0],VersionId=obj[1])
             try:
                 response = client.delete_bucket(Bucket=bucket_name)
             except ClientError as e:
